CODE_REVIEW.md

```diff
@@ -0,0 +1,17 @@
+**TASK 1: Code Review**
+* Not all the Reading list actions have respective reducers.
```

Comment:
Why do you think they should?

Reply:
As reducers are the functions that update state in NgRx. When user starts interacting on website an action gets triggered, we rely on the reducers to handle/update state as per the logic. So we need each actions to have respective reducers.

Reply:
This isn't strictly true. You can have actions that only trigger effects.